recipe/patches-grpc/0002-also-make-abseil-libs-private.patch

```diff
@@ -0,0 +1,37 @@
+From 8fb83860d176bf882bd97c8b9d48db6398298aad Mon Sep 17 00:00:00 2001
+From: "H. Vetinari" <h.vetinari@gmx.com>
+Date: Tue, 16 Aug 2022 12:30:08 +0200
+Subject: [PATCH 2/2] also make abseil libs private
+
+---
+ CMakeLists.txt | 14 +++++++-------
+ 1 file changed, 7 insertions(+), 7 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 844d42316c..d26718324a 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -2144,13 +2144,13 @@ target_link_libraries(grpc
+   PRIVATE ${_gRPC_RE2_LIBRARIES}
+   PRIVATE ${_gRPC_UPB_LIBRARIES}
+   PRIVATE ${_gRPC_ALLTARGETS_LIBRARIES}
+-  absl::flat_hash_map
+-  absl::inlined_vector
+-  absl::bind_front
+-  absl::hash
+-  absl::statusor
+-  absl::variant
+-  absl::utility
++  PRIVATE absl::flat_hash_map
++  PRIVATE absl::inlined_vector
++  PRIVATE absl::bind_front
++  PRIVATE absl::hash
++  PRIVATE absl::statusor
++  PRIVATE absl::variant
++  PRIVATE absl::utility
+   gpr
```

Comment:
Why don't these 3 lines need anything?

Reply:
Well, I started down this rabbit hole because the failures were quite clearly missing symbols from gpr, so I thought it might be okay to have them public since we actually ship those libs along with gprc. But trying that didn't solve the errors. Also, since it only appears on osx, it's still more likely that it's related to the `-framework` stuff.

But as I mentioned, I was mostly thrashing around, rather than doing something systematic